packages/i18n/src/pt.ts

```diff
@@ -0,0 +1,66 @@
+    ipv6:           (issue) => `IPv6 inválido: Recebido ${issue.received}`,
+    isoDate:        (issue) => `Data inválida: Recebido ${issue.received}`,
+    isoDateTime:    (issue) => `Data e hora inválidas: Recebido ${issue.received}`,
+    isoTime:        (issue) => `Hora inválida: Recebido ${issue.received}`,
+    isoTimeSecond:  (issue) => `Segundo inválido: Recebido ${issue.received}`,
+    isoTimestamp:   (issue) => `Timestamp inválido: Recebido ${issue.received}`,
+    isoWeek:        (issue) => `Semana inválida: Recebido ${issue.received}`,
+    length:         (issue) => `Tamanho inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    mac:            (issue) => `MAC inválido: Recebido ${issue.received}`,
+    mac48:          (issue) => `MAC de 48-bits inválido: Recebido ${issue.received}`,
+    mac64:          (issue) => `MAC de 64-bits inválido: Recebido ${issue.received}`,
+    maxBytes:       (issue) => `Bytes inválidos: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    maxLength:      (issue) => `Tamanho inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    maxSize:        (issue) => `Tamanho inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    maxValue:       (issue) => `Valor máximo inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    mimeType:       (issue) => `Tipo MIME inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    minBytes:       (issue) => `Bytes mínimos inválidos: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    minLength:      (issue) => `Tamanho inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    minSize:        (issue) => `Tamanho inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    minValue:       (issue) => `Valor inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    multipleOf:     (issue) => `Múltiplo inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    notBytes:       (issue) => `Bytes inválidos: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    notLength:      (issue) => `Tamanho inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    notSize:        (issue) => `Tamanho inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    notValue:       (issue) => `Valor inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    octal:          (issue) => `Octal inválido: Recebido ${issue.received}`,
+    regex:          (issue) => `Formato inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    safeInteger:    (issue) => `Inteiro seguro inválido: Recebido ${issue.received}`,
+    size:           (issue) => `Tamanho inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
+    startsWith:     (issue) => `Início inválido: Esperado ${issue.expected}, mas recebido ${issue.received}`,
```

Comment:
`Início inválido: Esperado ${issue.expected}, porém foi recebido ${issue.received}`